setup.py

```diff
@@ -102,4 +102,6 @@ def run(self):
     include_package_data=True,
     cmdclass={"build_ext": CMakeBuilder},
     zip_safe=False,
+    name="Mabrains_gdstk",
```

Comment:
Could you please remove both name and version from setup file, to be able to open PR for that change on main gdstk repo.